### PR TITLE
DataGrid Custom editors demo: Use svg instead of png to fix icons bluring in the lookup

### PR DESCRIPTION
**Pull request by MikeVitik**

### CODE REVIEW

File: JSDemos/images/icons/status-1.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="Слой_1" data-name="Слой 1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><title>status-1</title><path d="M8,2A6,6,0,1,1,2,8,6,6,0,0,1,8,2M8,0a8,8,0,1,0,8,8A8,8,0,0,0,8,0Z" style="fill:#b1b1b1"/><polygon points="9 4 7 4 7 9 11 9 11 7 9 7 9 4 9 4" style="fill:#b1b1b1"/></svg>
```
Comment (NickMitrokhin) [inline JSDemos/images/icons/status-1.svg]: We should not use russian words in our demos. Take a look at **id** and **data-name** attribures.

**Review by NickMitrokhin — APPROVED** [review]

File: JSDemos/Demos/PivotGrid/ExcelJSOverview/Angular/config.js [hunk 2]
```diff
@@ -1,71 +1 @@
-// In real applications, you should not transpile code in the browser. You can see how to create your own application with Angular and DevExtreme here:
-// https://js.devexpress.com/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/
-System.config({
```
Comment (vconst) [inline JSDemos/Demos/PivotGrid/ExcelJSOverview/Angular/config.js:3]: Why is this file modified?
